client/src/planwise/client/scenarios/views.cljs

```diff
@@ -139,7 +140,7 @@
       "leaflet-circle-for-change"))})
 
 (defn simple-map
-  [{:keys [bbox]} scenario state error read-only?]
+  [{:keys [bbox]} scenario state error read-only? unit-name]
```

Comment:
This extra parameter doesn't feel right. It's only passed in to forward to `show-provider`, but make little sense. The first parameter is the project, and the demand unit can be obtained from it. But better would be to obtain it through a subscription. It doesn't make much sense to forward it as an isolated parameter to `show-provider` as well. I'd either merge it with the provider. Needless to say the code needed a huge refactor before this change...

Reply:
Also, for these types of reagent components, don't forget to add mirror the arguments list in the returned function below (line 152)

Reply:
I don't know if it makes sense to have a subscription for something that will remain unchanged. I agree on extracting it on the simple-map function from the project instead of receiving it, but other solutions seem more complex and I don't see the benefit. 
Let's talk next week :)